docs/guides/embeds.md

```diff
@@ -0,0 +1,85 @@
+# How to Embed the Video.js player
+
+Video.js is meant to be an ehnacement to the video element in HTML5 so for years, it's embed code has been just a `<video>` element.
+Video.js then wraps the video element in a div that is used for us to place controls and anything else that's required for the player.
+
+For a long time this was enough. In 2016, a "div ingest" which allows the developer to give Video.js a player div to use instead of making it's own.
```

Comment:
Second sentence is incomplete. Maybe needs `was added` somewhere?

Reply:
oops, yes, something like that.